pyproject.toml

```
@@ -62,6 +62,7 @@ google = [ "google-cloud-speech~=2.32.0", "google-cloud-texttospeech~=2.26.0", "
 grok = []
 groq = [ "groq~=0.23.0" ]
 gstreamer = [ "pygobject~=3.50.0" ]
+heygen = [ "livekit>=0.19.1" ]
```

Is there a reason for this to be a different version than the livekit optional dep? Is this what heygen recommends?

Also, it looks like the `websockets` package is used.
```suggestion
heygen = [ "livekit>=0.19.1, websockets>=13.1,<15.0" ]
```

---

I was using the same one they used in their example, but there's no reason we can't stick with our default LiveKit voice.

---

done.